docs/contracts/index.md

```diff
@@ -60,6 +60,43 @@ The second, `check_contracts`, is a decorator allowing more fine-grained control
 .. autofunction:: python_ta.contracts.check_contracts(func_or_class)
 ```
 
+**Example: Disabling a single check**
```

Comment:
The content is good, but in terms of placement please create a new subsection called "Disabling checks" in between "Specifying contracts" and "Technical notes" (at the same heading level as those two) and move these examples into there. I would also add 1-2 introductory sentences explaining that these four arguments can be used to disable specific checks, etc.